components/button/src/index.js

```diff
@@ -23,18 +19,20 @@ const HALF_SHADOW = RAW_SHADOW / 2;
 const BASE_PAD = RAW_SPACING_2 - RAW_BORDER_WIDTH;
 
 const StyledButton = styled('button')(
-  ({ isStart }) => typography.font({
-    size: isStart ? 24 : 19,
-    lineHeight: isStart ? '1' : '19px',
-    weight: isStart ? 'bold' : undefined,
-  }),
+  ({ isStart }) =>
+    typography.font({
+      size: isStart ? 24 : 19,
+      lineHeight: isStart ? '1' : '19px',
+      weight: isStart ? 'bold' : undefined,
+    }),
   FOCUSABLE,
 
   ({
     buttonColour = BUTTON_COLOUR,
     buttonHoverColour = darken(0.05, buttonColour),
-    buttonShadowColour = (buttonColour === BUTTON_COLOUR)
-      ? BUTTON_COLOUR_DARKEN_15 : darken(0.15, buttonColour),
+    buttonShadowColour = buttonColour === BUTTON_COLOUR
```

Comment:
still not keen on the `?` being on the next line

to me when i'm reading this line i see `buttonShadowColour` being set to a boolean value